src/AddressRegistry.Projections.Extract/AddressExtract/AddressExtractProjectionsV2.cs

```diff
@@ -174,6 +174,27 @@ public AddressExtractProjectionsV2(IOptions<ExtractConfig> extractConfig, Encodi
                 UpdateVersie(item, message.Message.Provenance.Timestamp);
             });
 
+            When<Envelope<AddressWasRejectedBecauseHouseNumberWasRejected>>(async (context, message, ct) =>
```

Comment:
You are now handling this event twice. 
![image](https://user-images.githubusercontent.com/18366653/220359492-ef171bb3-40f6-4ce5-97ff-2544555d9770.png)